src/MakerInterface.php

```diff
@@ -53,7 +53,7 @@ public function configureDependencies(DependencyBuilder $dependencies): void;
      * @param ConsoleStyle   $io
      * @param Command        $command
      */
-    public function interact(InputInterface $input, ConsoleStyle $io, Command $command): void;
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command);
```

Comment:
This is too late. 1.0 is already released, and removing the return type of an interface is a BC break

Reply:
It's too late, unless we don't want it to be too late. We're not going to fork the bundle, neither to tag v2. Will we break anything if we bump to v1.0.1? Hell no. Let's be pragmatic.

Reply:
shipping a BC break in a patch release in an official SF bundle is not good: people would think it is OK to do such change in their own release, and this kind of defeat the BC promise (especially given that our BC documentation on http://symfony.com/doc/current/contributing/code/bc.html does not yet describe all changes regarding PHP 7 return types).

The *only* reason such BC break would have little impact here is because this interface was merged and released today, and so nobody had enough time to start using it. But this does not mean it is not a BC break.

Reply:
So answer about "will we break something" is yes: we will provide an official example that it is OK to do such change without considering it as a BC break, which impacts the usefulness of the BC rules for others (while we want them to follow them as well)

Reply:
OK, all things considered, I'm still :+1: (a HUGE ONE) to do the mode.
Now calling to others.